[Exercise-2] Otp-Field/src/components/OtpInputs.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {StyleSheet, View, TextInput} from 'react-native';
+import {Global, Colors, Spacing} from '../styles';
+
+const numRegex = /[0-9]/;
+
+const createRef = (inputs) => inputs.map(() => ({ref: React.createRef()}));
+
+class OtpInputs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      otp: createRef(props.inputs),
+    };
+  }
+
+  focusPrevious(key, index) {
+    const {otp} = this.state;
+    if (key === 'Backspace' && index !== 0) {
+      otp[index - 1].ref.current.focus();
+    }
+  }
+
+  focusNext(index, value) {
+    const {otp} = this.state;
+    const {getOtp} = this.props;
+    const isNumericValue = numRegex.test(value);
+    if (isNumericValue) {
```

Comment:
You can simply remove the nested if-else, like this
if(!isNumericValue) {
  return
}
...

Reply:
Done